topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+- [ ] Twitter - (@datacarpentry, @swcarpentry, @libcarpentry, @thecarpentries)
+- [ ] Blogs - (Data Carpentry, Software Carpentry, Library Carpentry, The Carpentries)
+- [ ] GitHub (carpentries, datacarpentry, swcarpentry, librarycarpentry, data-lessons, carpentrieslab)
+- [ ] Gitter - Library Carpentry Lobby
+- [ ] [Slack](https://swcarpentry.slack.com/messages)  
+- [ ] Facebook 
+- [ ] LinkedIn 
+
+Additionally, the community member will not be allowed to attend community discussions.
+
+### Teaching Workshops
+The suspended member will, for the duration of their suspension:
+- [ ] Be asked not to sign up for workshops on the Instructor Spreadsheet.
+- [ ] Not be able to participate as a helper at any workshops.
+- [ ] Be removed from Eventbrite login access.
+- [ ] Not be allowed to plan self-organized workshops.
+- [ ] Not be allowed to request or host centrally coordinated workshops.
+- [ ] Not be listed as an instructor on the “Instructors” webpage.
+- [ ] Be removed from the instructors email list.
+
+### Organizing Workshops 
+If the suspended member is a Workshop Administrator or serves in another role whereby they organize workshops for The Carpentries, for the duration of their suspension they will not have access to:
+- [ ] LastPass
+- [ ] AMY
+- [ ] Regional Email
+- [ ] Asana
+- [ ] HelpScout
+- [ ] Zoom
+- [ ] Workshop Administrator’s Slack Channel
+- [ ] Workshop Administrator’s Email List
+
+### Instructor Training
+The suspended member will, for the duration of their suspension, be removed from the “Our Trainers” web page. Additionally, the suspended member will not be allowed to: 
+- [ ] Teach Instructor Training or lead teaching demonstration sessions. The Deputy Director of Instructor Training (DDIT) will check the teaching demo Etherpad to ensure that the suspended member is not signed up to lead teaching demonstrations and will find a replacement Trainer as needed. 
+- [ ] Join Trainer meetings. The DDIT will check the meeting Etherpad, and if the suspended member is signed up to attend, will email them if needed to confirm they will not be in attendance. 
+- [ ] Post pull requests (PRs), issues, or comments to the Instructor Training GitHub repository. Write or administrative access to repositories will be removed for the duration of the suspension. 
+- [ ] Post to the Trainers Slack channel or mailing lists. The suspended member will be removed from these lists for the duration of their suspension. 
+- [ ] Participate in review of applications for open instructor training or Trainer training. 
+Participate as a learner in instructor training.
+
+### Trainer Training
+The suspended member will not be allowed to participate in Trainer Training (if they are currently enrolled) and will not be accepted for Trainer Training (if they apply during the suspension period). Additionally, they will not be allowed to shadow a current Carpentries Trainer. They may apply or reapply for future rounds of Trainer Training. 
```

Comment:
Considering how hard it is to get into instructor training, maybe this one should also be adjusted according to the seriousness of the situation? For things that might warrant a private reprimand, this seems like a disproportionate response.

Reply:
I see both sides here, but should we have someone who goes through this process and receives a suspension, perhaps this is the time to revisit their eligibility to become a trainer.  Leniency for someone reported or under review is great, but (IMO) trainers are expected to exhibit exemplary behavior as ambassadors and entry ways into our community.  

However, I'm absolutely find with this being something standard but open to discretion.

Reply:
This is something the CoCc can discuss and revise in the future.

Reply:
+1 to that.